base/bucket.go

```diff
@@ -127,10 +131,7 @@ func (spec BucketSpec) UseClientCert() bool {
 }
 
 func (spec BucketSpec) GetPoolName() string {
-	if spec.PoolName == "" {
-		return "default"
-	}
-	return spec.PoolName
+	return DefaultPool
```

Comment:
We don't have that many places that use GetPoolName() - seems cleaner to just replace those with base.DefaultPool directly?